### ISSUE TITLE
Need boot straight to home page in settings

### ISSUE THREAD

**Issue by ronyeh3** [yuliskov/SmartYouTubeTV#716]
Right now we can boot straight to subscription screen, watch later screen. If no user is signed in, a "sign in or skip" screen appears.

We need option to skip that screen permanently. And nvigate straight to home screen

**Comment by trulow** [comment]
I agree with this. Being able to skip the login page and go straight to the home screen would be awesome.
